File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/DriversSummaryResponse.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.DriversSummaryResponseSummaries
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param summaries 
 */
data class DriversSummaryResponse (

        @JsonProperty("Summaries") val summaries: List<DriversSummaryResponseSummaries>? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/DriversSummaryResponse.scala

package org.openapitools.client.model


case class DriversSummaryResponse (
    _Summaries: Option[List[DriversSummaryResponseSummaries]]
)
object DriversSummaryResponse {
    def toStringBody(var_Summaries: Object) =
        s"""
        | {
        | "Summaries":$var_Summaries
        | }
        """.stripMargin
}
